influxdb/tests/server_tests/influxdb_instance.py

```diff
@@ -175,6 +184,21 @@ def find_influxd_path(self):
 
         return influxdb_bin_path
 
+    def get_version(self):
+        """Get influxd version from stderr."""
+        regex = r".*v(\d\.\d\.\d).*"
+        p = re.compile(regex)
+        _cmd = self.influxd_path + ' version'
```

Comment:
This is being done twice (look at [this](https://github.com/influxdata/influxdb-python/blob/master/influxdb/tests/server_tests/influxdb_instance.py#L152)). Let's refactor that function to call out to your new one.

Also, can you please move to using `subprocess.check_output` to match existing code?